<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-L7E7ROD" failures="0" tests="5" name="batch.User_TC2" time="0.066" errors="1" timestamp="2023-04-06T12:52:58 IST" skipped="4">
  <testcase classname="batch.User_TC2" name="connectToDB" time="0.066">
    <error message="URLDecoder: Illegal hex characters in escape (%) pattern - Error at index 0 in: &quot;&quot;,&quot;" type="java.lang.IllegalArgumentException">
      <![CDATA[java.lang.IllegalArgumentException: URLDecoder: Illegal hex characters in escape (%) pattern - Error at index 0 in: "","
at java.base/java.net.URLDecoder.decode(URLDecoder.java:237)
at java.base/java.net.URLDecoder.decode(URLDecoder.java:147)
at com.mysql.cj.conf.ConnectionUrlParser.decode(ConnectionUrlParser.java:569)
at com.mysql.cj.conf.ConnectionUrlParser.parseConnectionString(ConnectionUrlParser.java:167)
at com.mysql.cj.conf.ConnectionUrlParser.<init>(ConnectionUrlParser.java:137)
at com.mysql.cj.conf.ConnectionUrlParser.parseConnectionString(ConnectionUrlParser.java:119)
at com.mysql.cj.conf.ConnectionUrl.getConnectionUrlInstance(ConnectionUrl.java:279)
at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:195)
at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:681)
at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:229)
at com.GenericUtilities.DatabaseUtility.connectToDb(DatabaseUtility.java:14)
at com.GenericUtilities.BaseClass.connectToDB(BaseClass.java:31)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:317)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- connectToDB -->
  <system-out/>
  <testcase classname="batch.User_TC2" name="searchProduct" time="0.000">
    <skipped/>
  </testcase> <!-- searchProduct -->
  <system-out>
    <![CDATA[product=======> test script execution started]]>
  </system-out>
  <testcase classname="batch.User_TC2" name="product" time="0.000">
    <skipped/>
  </testcase> <!-- product -->
  <system-out>
    <![CDATA[updateShipBillAddress=======> test script execution started]]>
  </system-out>
  <testcase classname="batch.User_TC2" name="updateShipBillAddress" time="0.000">
    <skipped/>
  </testcase> <!-- updateShipBillAddress -->
  <system-out>
    <![CDATA[payment=======> test script execution started]]>
  </system-out>
  <testcase classname="batch.User_TC2" name="payment" time="0.000">
    <skipped/>
  </testcase> <!-- payment -->
  <system-out/>
</testsuite> <!-- batch.User_TC2 -->
